benchmarks/transform_benchmark/specs.py

```
@@ -0,0 +1,182 @@
+def transform_specs(layout: str, spatial_size: int) -> list[BenchmarkSpec]:
+    """Return representative CPU-only and tensor-only transform cases."""
+    is_3d = layout in ("DHWC", "BDHWC")
+    axis = 1 if layout.startswith("B") else 0
+    crop_extent = max(8, spatial_size - spatial_size // 8)
+    crop_shape = (crop_extent, crop_extent, crop_extent) if is_3d else (crop_extent, crop_extent)
+    interpolation = ("trilinear", "nearest") if is_3d else ("bilinear", "nearest")
+    specs = [
+        BenchmarkSpec(
+            "NormalizeIntensity",
+            "cpu+gpu",
+            lambda l, s: NormalizeIntensity(keys=["image"], channel_wise=True, input_layout=l),
```

Ruff (0.16.2)

[error] Ambiguous variable name: `l`

(E741)